src/Block/Service/AbstractCategoriesBlockService.php

```diff
@@ -178,16 +179,16 @@ public function getMetadata(): MetadataInterface
     }
 
     /**
-     * @param CategoryInterface|int $id
-     * @param mixed                 $default
+     * @param CategoryInterface|int|string|null $id
```

Comment:
We need at least a CHANGELOG entry for added type hints

Reply:
They were already supported, the phpdoc was just wrong